tendrl/node_agent/discovery_modules/platform/plugins/centos.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import platform
+from tendrl.node_agent.discovery_modules.platform import base
+
+
+LOG = logging.getLogger(__name__)
+
+
+class CentosPlugin(base.PlatformBasePlugin):
```

Comment:
The name `PlatformBasePlugin` looks very generic. I feel we should call `PlatformDetectPlugin` or `PlatformDiscoverPlugin`